### DISCUSSION TITLE
AMP: Native AMP doesn't work in my Gigabyte Aorus Master RTX 3070 environment

### DISCUSSION THREAD

**Discussion by roma-glushko** [Lightning-AI/pytorch-lightning#7098] [General]
I'm trying to use AMP in my new Gigabyte Aorus Master RTX 3070 environment.
I have changed precision config from 32 to 16 and back to track global memory consumptions. Surprisingly, in both cases I see the same memory consumption (batch_size was the same).

Though, during training initialization, PyTorch Lightning seems to recognise that AMP should be used for the training:
<img width="1838" alt="Screenshot 2021-04-19 at 16 00 23" src="https://user-images.githubusercontent.com/9402690/115240672-a68db280-a128-11eb-9d57-be2d3e73fdc5.png">

Despite that, memory consumptions are the same:

<img width="726" alt="Screenshot 2021-04-19 at 16 04 43" src="https://user-images.githubusercontent.com/9402690/115241016-01270e80-a129-11eb-8e71-53705001e3d2.png">

I expect AMP to reduce my memory consumptions.

My dependencies:
- PyTorch was installed this way:

```bash
pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 -f https://download.pytorch.org/whl/torch_stable.htm
```
- PyTorch Lightning: 1.2.3
- Python: 3.8.6

Any clues why it does work for me?

**Comment by roma-glushko** [comment]
Hey @Borda @williamFalcon 👋 have you seen anything similar to this issue?

**Comment by roma-glushko** [comment]
I had a conversation with PyTorch community about this issue to exclude a case of environment misconfiguration on my side (https://discuss.pytorch.org/t/rtx-3070-amp-doesnt-seem-to-be-working/118835/7)

We came up to a conclusion that, generally, AMP works there, but there is a project, that I originally mentioned, which possibly has a combination of model/dataset that triggers an issue on Lightning side.

**Reply by awaelchli** [reply]
if you can share a small code sample that shows the problem on the latest Lightning version, ideally using our debugging template, and post it as a bug report in github then we can look into the issue.
